Specs/Core/buildModuleUrlSpec.js

```diff
@@ -1,13 +1,15 @@
 import { buildModuleUrl } from "../../Source/Cesium.js";
 import { Resource } from "../../Source/Cesium.js";
-import { Uri } from "../../Source/Cesium.js";
+import { urijs as URI } from "../../Source/Cesium.js";
 
 describe("Core/buildModuleUrl", function () {
   it("produces an absolute URL for a module", function () {
     var url = buildModuleUrl("Workers/transferTypedArrayTest.js");
 
     expect(url).toMatch(/Workers\/transferTypedArrayTest.js$/);
-    expect(new Uri(url).isAbsolute()).toBe(true);
+    var uri = new URI(url);
+    expect(uri.scheme().length).toBeGreaterThan(0);
```

Comment:
`urijs` does not have an equivalent `isAbsolute()` function so I checked the scheme and fragment manually per the old Uri.js implementation. This change happens again in `TaskProcessor`.